flashcard_model.rb

```diff
@@ -0,0 +1,60 @@
+require 'pry'
+
+class FlashcardModel
+  attr_reader :file, :answers, :parsed_file
+
+  def initialize(file)
+    @file = file
+    @answers = []
+    @parsed_file = []
+    create_nested_array
+  end
+
+  def array_creator
+    txt = File.open(file, "r+")
+    file_array = txt.map do |row|
+      if row == "\n"
+        row = false
+      end
+        row
+    end
+    file_array
+  end
+
+  def array_chomper
+    file_array = array_creator
+    clean_file_array = file_array.map do |row|
+      if row.class == String
+        row = row.chomp
+      end
+      row
+    end
+    clean_file_array
+  end
```

Comment:
I like your dedication to single responsibility methods. But I don't think you need to go this far. I would just add chomping to your iteration in create_nested_array. Iteration is expensive so try not to iterate over something multiple times if you don't have to.

While it's not such a big deal with the amount of flash cards you have, a good habit to get into is imagining that your dataset is something ridiculous, like 3 trillion flashcards. Iterating twice would take a lot longer.